x-pack/plugin/watcher/src/test/java/org/elasticsearch/xpack/watcher/notification/NotificationServiceTests.java

```diff
@@ -3,7 +3,7 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-package org.elasticsearch.xpack.notification;
+package org.elasticsearch.xpack.watcher.notification;
```

Comment:
I had to do this to use the package private constructor... also it was the only test *not* under notification :)